bot/utility/utility.py

```diff
@@ -206,6 +208,50 @@ async def welcome_message(self, user: discord.Member):
                                                               "__gemeinsam__ schaffen wir das!** :muscle:")
         await user.send(content=content, embed=embed)
 
+    @commands.command(name='format')
+    async def format(self, ctx: commands.Context, language: Optional[str], *code: str):
+        """Handler for the `format` command. Formats the provided message using clang-format and prints it using markdown syntax highlighting
+            Refer to the !howto format for an extended explanation of how to use it
+        Args:
+            language: The language abbreviation to highlight the resulting code with
+
+        """
+        try:
+            if language is None:
+                raise Exception("No language provided")
```

Comment:
If language is typed as an Optional[str], then None should be a valid value and not raise an exception. 

Further, invalid user input should not raise exceptions since it's not exceptional but expected behavior. 
A simple error answer should be sufficient and no exception should be thrown.